Artifacts/windows-azuredevops-multiple-agents/run.ps1

```diff
@@ -0,0 +1,122 @@
+# Downloads the Azure DevOps Pipelines Agent and installs specified instances on the new machine
+# under C:\agents\ and registers with the Azure DevOps Pipelines agent pool
+
+# Enable -Verbose option
+[CmdletBinding()]
+Param
+(
+	[Parameter(Mandatory=$true)]
```

Comment:
Don't make these `[Parameter(Mandatory=$true)]`. Just validate within your script and `throw` accordingly. The reason is that if a user mistakenly passes no values, without knowing and if allowed, then the script will hang, waiting for input.